### DISCUSSION TITLE
Per country leaderboard

### DISCUSSION THREAD

**Discussion by vycdev** [monkeytypegame/monkeytype#2059] [Ideas]
It would be cool if we could set our country in the profile, and get per country leaderboards (of course that doesn't mean removing the global leaderboard).

**Comment by frogfrogfrogfrogfrog** [comment]
I feel like that would be adding way too many leaderboards though. Since you'd have to include every country so that one country wouldn't be left out. It also would be very biased towards the US since I figure most users are in that country.

**Reply by vycdev** [reply]
Coming back after like 2 years, because I still think per country leaderboards should be a thing. Bias towards a country doesn't really matter, the idea is to see where you position yourself in your own country. Also the number of countries is pretty much static, new countries don't form every day and its only about 200 countries in total. Tetrio does the per country leaderboards thing and i think it works pretty nicely.

**Reply by prescientmoon** [reply]
Sounds like an awesome idea

**Comment by matru** [comment]
I think the idea of introducing countries is pretty cool. But there doesn't need to be a separate leader board for it, just let people choose which country they want to represent (have an option to choose something fictitious like Azeroth or a blank options etc). You can even setup an option for a custom representation, but I think limiting it exclusively to countries (with few additional selections outside the real world) should be the right path in order to encourage people actually wanting to represent their country (or even just a country they love enough to put it's badge on their profile) and motivate themselves to push for more, as explained down below.

In addition, you can have a hidden per country ladder, when they choose a country they want to represent, on their profile (important: viewable only to themselves) they can find their rank in the country (a selection for viewing for the day, or all time might not be a bad idea, but if I were to pick one it would be all-time). Importantly once again, this should not be a ranked list with all of the people representing the given country, but just a simple personal rank placement number that will be only privately viewable once you log in to your account. This can encourage people to actually rank up and compare themselves in the country they've picked, while also discouraging people so they won't pick countries that are the least competitive just for showing off and saturating those countries.

Now I am not sure if you can scroll down manually and scrape the leader board data all the way to the last person on it, but this is still a big hurdle one needs to go through and it doesn't really achieve much.

Lastly you can lock in changing countries to once every 30 days, or something similar to further discourage people for bouncing countries just they can.

**Reply by dawkaka** [reply]
It doesn't have to be a separate leaderboard, just a filter. the same leaderboard but should be able to filter by country.
